optimizely/project_config.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, datafile, logger, error_handler):
         """
 
         config = json.loads(datafile)
-        self._datafile = datafile
+        self._datafile = str(datafile)
```

Comment:
You will have to specify encoding here if I understand correctly. It should be `utf-8`.

Python 2 by default does `ascii` and Python 3 does `utf-8`.